packages/sdk/lib/base.ts

```diff
@@ -99,7 +99,7 @@ export abstract class Plugin {
     validationResponses: ValidationResponse[],
   ): void {}
 
-  track(userId: string | undefined, event: Event, options?: TrackOptions): void {}
+  track(userId: string | undefined, event: Event, options?: unknown): void {}
```

Comment:
Is `unknown` here necessary to allow the Plugin implementations to type cast to their own <Plugin>TrackOptions?

Reply:
`any` should work too